From b75e2fb3a4e873ac3ed0856aa78d34f0ba5eeae3 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Sun, 5 Nov 2017 10:47:31 +0200
Subject: [PATCH 03/51] ARM: dts: imx7d: cl-som-imx7: fix i2c2 configuration

Fix I2C2 pinmux configuration: change pins and move to the
iomuxc_lpsr node.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/boot/dts/imx7d-cl-som-imx7.dts | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
index bb58a76..6cb382c 100644
--- a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
+++ b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
@@ -247,13 +247,6 @@
 		>;
 	};
 
-	pinctrl_i2c2: i2c2grp {
-		fsl,pins = <
-			MX7D_PAD_I2C2_SDA__I2C2_SDA		0x4000007f
-			MX7D_PAD_I2C2_SCL__I2C2_SCL		0x4000007f
-		>;
-	};
-
 	pinctrl_uart1: uart1grp {
 		fsl,pins = <
 			MX7D_PAD_UART1_TX_DATA__UART1_DCE_TX		0x79
@@ -279,6 +272,13 @@
 };
 
 &iomuxc_lpsr {
+	pinctrl_i2c2: i2c2grp {
+		fsl,pins = <
+			MX7D_PAD_GPIO1_IO07__I2C2_SDA			0x4000000f
+			MX7D_PAD_GPIO1_IO06__I2C2_SCL			0x4000000f
+		>;
+	};
+
 	pinctrl_usbotg1: usbotg1grp {
 		fsl,pins = <
 			MX7D_PAD_GPIO1_IO05__GPIO1_IO5			0x14 /* OTG PWREN */
-- 
1.9.1

